libs/output-mapping/src/DeferredTasks/FailedLoadTableDecider.php

```diff
@@ -21,9 +21,14 @@ public static function decideTableDelete(
             // likely the table doesn't exist, but any other error really prevents us from positive decision
             return false;
         }
+        $metadata = $tableInfo['metadata'];
+        if ($tableInfo['isTyped']) {
+            $metadata = array_filter($metadata, fn($m) => $m['provider'] !== 'storage');
```

Comment:
Já bych preventivně vyfiltroval pryč jen jedno konkrétní `KBC.dataTypesEnabled` od providera `storage`

Reply:
jj to by šlo - https://github.com/keboola/platform-libraries/pull/309/commits/6281585a12415c06b5a7bd91479f7f401965305f